src/views/Options.vue

```diff
@@ -220,11 +223,36 @@ export default {
   data: () => ({
     passwordDialog: false
   }),
+  mounted() {
+    nextTick(() => {
+      if (this.sidebarLayoutRef && this.scrollTopPosition) {
+        this.sidebarLayoutRef.scrollTo({
+          top: this.scrollTopPosition
+        })
+      }
+    })
+  },
+  beforeUnmount() {
+    if (this.sidebarLayoutRef) {
+      this.scrollTopPosition = this.sidebarLayoutRef.scrollTop || 0
+    }
+  },
   computed: {
     className: () => 'settings-view',
     stayLoggedIn() {
       return this.$store.state.options.stayLoggedIn
     },
+    scrollTopPosition: {
+      get() {
+        return this.$store.state.options.scrollTopPosition
+      },
+      set(value) {
+        this.$store.commit('options/updateOption', {
+          key: 'scrollTopPosition',
+          value
+        })
+      }
+    },
```

Comment:
Can we move the scroll position logic into a hook and reuse it in every View where it's needed? @S-FrontendDev